webapp/src/components/Modal/AuctionModal/AuctionModal.js

```diff
@@ -88,8 +124,6 @@ export default class AuctionModal extends React.PureComponent {
   }
 
   renderWelcome() {
-    const { hasAgreedToTerms } = this.state
```

Comment:
_Silly comment_ We're using `this.state.` a few times below, maybe destructuring works here?

Reply:
this is because `hasAgreedToMainTerms` can't be deconstructed cos there's a function with that name